sound/soc/sof/intel/hda-dsp.c

```diff
@@ -455,6 +455,11 @@ int hda_dsp_set_hw_params_upon_resume(struct snd_sof_dev *sdev)
 			if (!link)
 				return -EINVAL;
 			stream_tag = hdac_stream(stream)->stream_tag;
+
+			if (hdac_stream(stream)->direction ==
+				SNDRV_PCM_STREAM_CAPTURE)
+				continue;
+
```

Comment:
Why not use the same format:
```
	if (substream->stream == SNDRV_PCM_STREAM_PLAYBACK)
		snd_hdac_ext_link_clear_stream_id(link, stream_tag);
```

Reply:
Because of limitation of 80 char. With the line of if,  
the size of line "snd_hdac_ext_link_clear_stream_id(link, stream_tag);"  is also more than 80 chars.
to make it like:
snd_hdac_ext_link_clear_stream_id(link,
                                                           stream_tag);
It is also more than 80 chars.

I did this is to make it beautiful